simulations/vip-416/opbnbmainnet.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from "chai";
+import { Contract } from "ethers";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, pretendExecutingVip, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import vip021, { COMPTROLLERS, NTGs, PSR, VTOKENS, XVS_STORE } from "../../multisig/proposals/opbnbmainnet/vip-024";
+import vip416, { OPBNBMAINNET_BOUND_VALIDATOR, OPBNBMAINNET_XVS_BRIDGE_ADMIN } from "../../vips/vip-416/bscmainnet";
+import OWNERSHIP_ABI from "../vip-416/abi/Ownership.json";
+
+const XVS_BRIDGE = "0x100D331C1B5Dcd41eACB1eCeD0e83DCEbf3498B2";
+const { opbnbmainnet } = NETWORK_ADDRESSES;
+
+forking(45875653, async () => {
+  const provider = ethers.provider;
+  let binanceOracle: Contract;
+  let resilientOracle: Contract;
+  let boundValidator: Contract;
+  let treasury: Contract;
+  let xvsBridgeAdmin: Contract;
+  let xvsBridge: Contract;
+  const xvsVaultProxy = new ethers.Contract(opbnbmainnet.XVS_VAULT_PROXY, OWNERSHIP_ABI, provider);
+  const xvsStore = new ethers.Contract(XVS_STORE, OWNERSHIP_ABI, provider);
+
+  before(async () => {
+    binanceOracle = new ethers.Contract(opbnbmainnet.BINANCE_ORACLE, OWNERSHIP_ABI, provider);
+    resilientOracle = new ethers.Contract(opbnbmainnet.RESILIENT_ORACLE, OWNERSHIP_ABI, provider);
+    boundValidator = new ethers.Contract(OPBNBMAINNET_BOUND_VALIDATOR, OWNERSHIP_ABI, provider);
+    xvsBridgeAdmin = await ethers.getContractAt(OWNERSHIP_ABI, OPBNBMAINNET_XVS_BRIDGE_ADMIN);
+    xvsBridge = await ethers.getContractAt(OWNERSHIP_ABI, XVS_BRIDGE);
+    treasury = await ethers.getContractAt(OWNERSHIP_ABI, opbnbmainnet.VTREASURY);
+    await pretendExecutingVip(await vip021());
+  });
+
+  testForkedNetworkVipCommands("vip333 XVS Bridge permissions", await vip416());
```

Comment:
Review every test used in the `testForkedNetworkVipCommands` functions. I would avoid specific references to VIP numbers, to simplify the refactors in the future. For example, in this case, we could simply say "Accept ownerships/admins"